UI/window-projector.cpp

```diff
@@ -438,12 +429,14 @@ void OBSProjector::OBSRenderMultiview(void *data, uint32_t cx, uint32_t cy)
 		}
 	};
 
-	/* ----------------------------- */
-	/* draw sources                  */
-
```

Comment:
I'd probably not recommend removing these sort of code segmentation comments.  These comments were put in here to make a larger function like this a bit more readable.